### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by alexstorm1** [jmalarcon/Win11ClockToggler#20]
Hi, great tool, I've been looking for something like this for a while now!
Just one major question: could you please somehow implement a "start on reboot" with it automatically enable itself and minimize the window?
I tried using the CLI version by storing a shortcut at shell:startup with "-s -b" for this, but its a bit buggy and also does not support the hovering feature! Thanks in advance!

The CLI version works as intended for this until you hover over the main monitors area where the clock should be. It then appears and never goes away, but only for the main monitor. Im using 3 monitors. So either a GUI version with a "start on reboot" feature or a CLI version with hover feature support and a solution for the bug would be awesome. Thanks!

**Comment by jmalarcon** [comment]
Hi:

Thanks for the idea. However this is not a bug, is simply a missing feature.

I designed the app to have an easy way to hide the clock and notification areas tempraryly

**Comment by jmalarcon** [comment]
Hi:

Thanks for the idea. However this is not a bug, is simply a missing feature.

I designed the app to have an easy way to hide the clock and notification areas temporarily for recording videos (which is what I do a lot) and the like, but not as a mean to have the clock hidden by default. I'll think about a way to do it if I have the time, but I can't promise.

The CLI version works as good as possible taking into account that it's not a resident app. I mean, as soon as it hides or shows the elements it ends and has no influence in the system. Unfortunately Windows automatically re-shows the areas when specific events happen (i.e.: a new element is added to the notification area or when a new notification pops up), and the CLI version can't do a thing about it as is explained in the readme file. In fact in the readme I explain a way to prevent this (mostly) happening.

Summing up: it's not a bug, is different orientation for the app, I appreciate the idea and I'll think about it in the future.

**Comment by alexstorm1** [comment]
I understand, then take it as a suggestion. An auto start on reboot, with it enabling itself and closing the window itself, would be awesome for the GUI version. Hope you'll find some time to implement it.
